common/src/paused_program_storage.rs

```diff
@@ -80,4 +96,81 @@ pub trait PausedProgramStorage: super::ProgramStorage {
 
         Ok(gas_reservations)
     }
+
+    /// Resume program with the given key `program_id`.
+    fn resume_program(
+        program_id: ProgramId,
+        allocations: BTreeSet<WasmPage>,
```

Comment:
Why do you use btreeset + MemoryMap and not just single sorted list of `(PageNumber, PageContent)` ?

Reply:
changed according to refactoring